db/migrate/20220317235629_create_testimonials.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    create_table :testimonials do |t|
+      t.varchar :name, NULL: false
+      t.varchar :content, NULL: true
```

Comment:
Please check what datatypes are allowed by `active record`, and `null: true` is redundant in this case because is the default setting.